### PR TITLE
Added changes for as per API upgrade

### PR DESCRIPTION
**Pull request by KarishmaBothara**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,11 +44,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.5",
-    "@cennznet/api": "^0.14.1",
-    "@cennznet/crml-cennzx-spot": "^0.6.1",
-    "@cennznet/crml-generic-asset": "^0.9.1",
-    "@cennznet/types": "^0.14.1",
-    "@cennznet/util": "^0.14.1",
+    "@cennznet/api": "^0.20.4",
```
Comment (ianhe8x) [inline package.json]: use v0.20.6 which includes several bug fixes

File: src/injection/__mocks__/signOnSingleSource.ts [hunk 2]
```diff
@@ -19,27 +12,23 @@ const vault =
   '0x4a0140a515857eb9c05399385795738cc8cf898d12fcd982c06c4c034fed48e46c520bdfe02980c2c745572c68c5c50bc1c0126acc5c6565d1e6a226582fcb3330ef4f4b87aed90cb7d7cd6176ac1c22c792ed5327ed91df6476e4a13b8441ede0d364fe5488a655ffe25afdc4fa9d1c32258d719124263f51525feb21a09d3d61d9312ce96c1483f562496d4b106dc87115ed4bd77e502a068b63332702bf77267640be0c94a415beb0357fba52e0e050e7ffafbc884d13ea5577ddcfcab54aa8e79b28e78c2f16c2af715b60a737332625e5';
 
 // @ts-ignore
-signOnSingleSource = async (payload: SignPayload) => {
+signOnSingleSource = async (payload: SignerPayloadJSON) => {
   const { method, meta, address, blockHash, nonce, era, version } = payload;
 
   const options = {
```
Comment (ianhe8x) [inline src/injection/__mocks__/signOnSingleSource.ts]: where's this options used?

File: src/injection/__tests__/signer.test.ts [hunk 3]
```diff
@@ -4,15 +4,13 @@ global.window = {};
 console.log = () => {};
 console.error = () => {};
 
-import { Api } from '@cennznet/api';
-import { WsProvider } from '@cennznet/api/polkadot';
-import { GenericAsset } from '@cennznet/crml-generic-asset';
+import { Api, WsProvider } from '@cennznet/api';
```
Comment (ianhe8x) [inline src/injection/__tests__/signer.test.ts]: WsProvider is not exported in `@cennznet/api` >0.20.5

File: src/injection/__tests__/signer.test.ts [hunk 4]
```diff
@@ -25,17 +23,14 @@ let api: Api = null;
 
 describe('encode & decode extrinsic', () => {
   beforeAll(async () => {
-    const provider = new WsProvider(
-      'wss://cennznet-node-0.centrality.cloud:9944'
-    );
-    api = await Api.create({ provider });
+    api = await Api.create({ provider: 'wss://cennznet-node-0.centrality.cloud:9944' });
```
Comment (ianhe8x) [inline src/injection/__tests__/signer.test.ts]: change to 'wss://rimu.unfrastructure.io/public/ws'

**Review by ianhe8x — CHANGES_REQUESTED** [review]

File: src/injection/__mocks__/signOnSingleSource.ts (hunk 2 above)
Comment (KarishmaBothara) [reply src/injection/__mocks__/signOnSingleSource.ts]: not needed.

**Review by ianhe8x — APPROVED** [review]
lgtm

**Pull request merged**